services/movie.py

```diff
@@ -0,0 +1,38 @@
+from django.db.models import QuerySet
+
+from db.models import Movie
+
+
+def get_movies(genres_ids: list = None, actors_ids: list = None) -> QuerySet:

```

Comment:
You should specify inner type of list and QuerySet. If argument can be None, you should show it in type hint. Fix everywhere, please
```suggestion
def get_movies(genres_ids: list[int] | None = None, actors_ids: list[int] } None = None) -> QuerySet[Movie]:

```